### PR TITLE
chore: Add issue templates

### PR DESCRIPTION
**Pull request by szuperaz**
Based on: https://github.com/GetStream/sdk-template/tree/main

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+**Version**
+
+- SDK version
+
+**Env:**
+
+- OS: [e.g. iOS, linux...]
+- Browser [e.g. chrome, safari]
+- Emulator [e.g. android emulator, real phone...]
```
Comment (khushal87) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]: `Device:`

We should also preferably add device here.

Comment (oliverlaz) [reply .github/ISSUE_TEMPLATE/bug_report.md:40]:
```suggestion
- Device: [e.g. android emulator, real phone...]
```
Yeah, I also think we should replace `Emulator` with `Device` here.

**Review by oliverlaz — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (szuperaz) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Makes sense, I've also separated env info based on packages, please check: https://github.com/GetStream/stream-video-js/pull/846/commits/e175296540204ddd87045b563c39f59c1d2b31e1

Comment (khushal87) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Perfect! 😄

**Pull request merged**